src/features/lightspeed/api.ts

```diff
@@ -176,6 +177,12 @@ export class LightSpeedAPI {
     }
     const rhUserHasSeat = await this.lightSpeedAuthProvider.rhUserHasSeat();
 
+    const userProvidedModel =
+      lightSpeedManager.settingsManager.settings.lightSpeedService.model;
```

Comment:
It's a bit of complexity just to avoid the situation where the model is undefined or empty. What the problem if you send an empty model value? Something like this:

```
inputData.model = lightSpeedManager.settingsManager.settings.lightSpeedService.model;
```

Reply:
that's true!! I was just copy pasting from another place, but that's true, no need for that complexity. Fixed. Thanks!